website/src/components/comment-section.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { ReactNode, SetStateAction } from "react"
+import { useState } from "react"
+import * as Dailp from "src/graphql/dailp"
+import { TranslatedParagraph } from "src/segment"
+import * as css from "./comment-section.css"
+
+export const CommentSection = (p: {
+  parent: Dailp.FormFieldsFragment | TranslatedParagraph
+}) => {
+  return (
+    <div>
+      {p.parent.__typename === "DocumentParagraph" ? (
+        <ParagraphCommentSection paragraph={p.parent} />
+      ) : p.parent.__typename === "AnnotatedForm" ? (
+        <WordCommentSection word={p.parent} />
+      ) : (
+        ""
+      )}
+    </div>
+  )
+}
+
+export const WordCommentSection = (p: { word: Dailp.FormFieldsFragment }) => {
+  const [{ data }] = Dailp.useWordCommentsQuery({
+    variables: { wordId: p.word.id },
+  })
+
+  const wordComments = data?.wordById
```

Comment:
if youre naming this `wordComments`, then maybe it should hold the actual list of comments and not the word info.

```
let wordComments = data?.wordById?.comments
```

Then line 32 would read

```
wordComments?.map(...)
```